homeassistant/components/here_travel_time/strings.json

```diff
@@ -60,7 +60,7 @@
     "step": {
       "init": {
         "data": {
-          "traffic_mode": "Traffic mode",
+          "traffic_mode": "Use traffic and time-aware routing. (Needed for defining arrival/departure times)",
```

Comment:
Let's add the thing behind parenthesis in `data_description` as that feels like a better place